pkg/ccl/backupccl/backup_test.go

```diff
@@ -9904,7 +9930,12 @@ func TestBackupRestoreTelemetryEvents(t *testing.T) {
 	defer jobs.TestingSetProgressThresholds()()
 
 	baseDir := "testdata"
-	args := base.TestServerArgs{ExternalIODir: baseDir, Knobs: base.TestingKnobs{JobsTestingKnobs: jobs.NewTestingKnobsWithShortIntervals()}}
+	args := base.TestServerArgs{
+		ExternalIODir: baseDir,
+
+		// Disabled to run within tenant because test can panic
+		DisableDefaultTestTenant: true,
```

Comment:
Can we file an issue, and mention it here so that we investigate it outside this PR.